scripts/install/setup_wsl2.sh

```
@@ -0,0 +1,45 @@
+    # cargoのパス設定
+    [[ -f "$HOME/.cargo/env" ]] && source "$HOME/.cargo/env"
+
+    # ツールのインストール
+    install_if_missing "starship" "curl -sS https://starship.rs/install.sh | sh -s -- --yes"
+    install_if_missing "sheldon" "cargo install sheldon"
+    install_if_missing "eza" "cargo install eza"
+
+    # zshをデフォルトシェルに設定
+    if [[ "$SHELL" != *"zsh"* ]]; then
+        info "zshをデフォルトシェルに設定中..."
+        chsh -s $(which zsh)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)